README.md

```diff
@@ -75,27 +75,11 @@ Note that if the user’s session expires before this timeout (e.g. a short `ses
 
 ### Garbage collection
 
-By default, this module will trigger garbage collection of expired sessions once in every 50 requests (approximately, using `mt_rand()`). It’s advised to disable this method of garbage collection and instead set up a cron task which points to `dev/tasks/LoginSessionGarbageCollectionTask`.
+By default, this module will trigger garbage collection of expired sessions once every 5 minutes through a cron task.
```

Comment:
I think this section could use rewording (per my other suggestion that the crontask module shouldn't be mandated.)

---

Expired sessions need to be cleaned up periodically to avoid bloating the database. There are two methods available to manage this:

#### Via `silverstripe/crontask` (recommended)

If you have the [`silverstripe/crontask` module](https://github.com/silverstripe/silverstripe-crontask) installed and [configured](https://github.com/silverstripe/silverstripe-crontask#server-configuration), garbage collection will run automatically every 5 minutes via `GarbageCollectionCronTask`, and no further action is required.

#### Via `LoginSessionGarbageCollectionTask`

Alternatively, you can create a system cron entry to run the `LoginSessionGarbageCollectionTask` directly on a regular cadence:

```
*/5 * * * * /path/to/webroot/vendor/bin/sake dev/tasks/LoginSessionGarbageCollectionTask
```